ckan/cli/__init__.py

```diff
@@ -5,7 +5,7 @@
 import click
 import logging
 from logging.config import fileConfig as loggingFileConfig
-from configparser import ConfigParser
+from six.moves.configparser import ConfigParser
```

Comment:
Not really sure how this is working for anyone else? This looks to be way that this library is imported in other parts of the project.